### DISCUSSION TITLE
Integrate PhotoMaker

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#1945] [Ideas]
https://github.com/TencentARC/PhotoMaker
https://huggingface.co/spaces/TencentARC/PhotoMaker
https://youtu.be/NWIdzTEk5O4

**Comment by xhoxye** [comment]
Hopefully Fooocus can also use this PhotoMaker

**Reply by BassAzayda** [reply]
This!

**Comment by Irmagon** [comment]
Read this: https://github.com/lllyasviel/Fooocus/issues/1980#issuecomment-1900062476
